amethyst_core/src/lib.rs

```diff
@@ -13,6 +13,7 @@ pub use self::{
     axis::{Axis2, Axis3},
     event::EventReader,
     hidden::{Hidden, HiddenPropagate},
+    hide_hierarchy_system::build,
```

Comment:
I don't think this is the namespacing we want to use, probably can remove `::build`